pkg/acsengine/const.go

```diff
@@ -85,6 +85,10 @@ const (
 	DefaultOrchestratorName = "k8s"
 	// DefaultEtcdDiskSize specifies the default size for Kubernetes master etcd disk volumes in GB
 	DefaultEtcdDiskSize = "128"
+	// DefaultReschedulerImage defines the rescheduler deployment version on Kubernetes Clusters
+	DefaultReschedulerImage = "rescheduler:v0.3.1"
```

Comment:
Is the right version to use as default?

Reply:
let's ship w/ latest

Reply:
0.3.1 is the latest tag [here](https://console.cloud.google.com/gcr/images/google-containers/GLOBAL/rescheduler)